binstar_client/commands/login.py

```diff
@@ -77,8 +78,13 @@ def interactive_get_token(args, fail_if_already_exists=True):
 
 
     if token is None:
-        msg = ('Sorry. Please try again '
-               '(go to https://anaconda.org/account/forgot_password '
+        if parsed_url.netloc.startswith('api.anaconda.org'):
+            netloc = 'anaconda.org'
+        else:
+            netloc = parsed_url.netloc
```

Comment:
In this case `parsed_url.netloc` will be something like `example.com/api` the `/api` should be removed.